### PR TITLE
add search function and content on the page for the diseases

### PR DESCRIPTION
**Pull request by cristinamerisoiu**
added search function, created content for the diseases. adjusted some of the design.

### CODE REVIEW

File: src/Globalstyles.js [hunk 1]
```diff
@@ -10,9 +10,8 @@ body {
   font-size: 16px;
   margin: 0;
   font-family: 'Roboto', sans-serif;
-  width:100vw;
-  height:100vh;
-  background:#E8F1F2; 
-  position:fixed;
+  width:100%;
+  height:100%;
+  background:#E8F1F2;
```
Comment (webdevelukas) [inline src/Globalstyles.js]: You use this background-color also in CardContainer.js. Maybe you can make a variable out of it?

File: src/components/SearchVaccines.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  opacity: 0.5;
+  margin-top: 10px;
+  text-align: center;
+
+  &::placeholder {
+    color: grey;
+  }
+`;
+
+export default function SearchVaccines({ onSearch }) {
```
Comment (webdevelukas) [inline src/components/SearchVaccines.js]: Sounds like a function but not like a component to me.

File: src/Globalstyles.js (hunk 1 above)
Comment (cristinamerisoiu) [reply src/Globalstyles.js]: Yes, i have to think though if I need the CardContainer. I don't think I will need it anymore. I have to see how things develop.

File: src/components/SearchVaccines.js (hunk 2 above)
Comment (webdevelukas) [reply src/components/SearchVaccines.js]: SearchBar sounds like a component for example.

Comment (cristinamerisoiu) [reply src/components/SearchVaccines.js]: done :)

**Pull request merged**